bcl/inc/bc_module_rs485.h

```diff
@@ -0,0 +1,135 @@
+#ifndef _BC_MODULE_RS485_H
+#define _BC_MODULE_RS485_H
+
+#include "bc_tick.h"
+#include "bc_sc16is740.h"
+#include "bc_scheduler.h"
+#include "bc_fifo.h"
+
+//! @addtogroup bc_module_rs485 bc_module_rs485
+//! @brief Driver for RS-485 Module
+//! @{
+
+//! @brief Callback events
+
+typedef enum
+{
+    //! @brief Error event
+    BC_MODULE_RS485_EVENT_ERROR = 0,
+
+    //! @brief Update event
+    BC_MODULE_RS485_EVENT_VOLTAGE = 1,
+
+    //! @brief Async write done
+    BC_MODULE_RS485_EVENT_ASYNC_WRITE_DONE = 2,
+
+    //! @brief Event is reading done
```

Comment:
Looking above on "Error event", it seems better convention as "Reading done event", so you can avoid "is" verb, hm?